### PR TITLE
Fix dependency errors

### PR DESCRIPTION
**Pull request by ajtadeo**
tldr; the packages for this LL are very outdated since it hasn't been updated in a while + some circular dependencies that wouldn't allow me to uninstall packages, so i ended up removing the package.json file to install everything from scratch and update old code to work with the updated packages

### Changes
* removed `package.json` and re-ran `npm init` to generate new `package.json` and `package-lock.json` files
* installed all previously installed packages using react v17
* updated react router code to v6 https://reactrouter.com/en/main/upgrading/v5
  * Switch --> Routes
  * useHistory --> useNavigation
* updated bulma framework import in `App.scss`. this also made me remove all the semicolons in this file and idk why, this might be indicative of another bug going on
* in `Bake.scss`, updated all `hsl()` functions to include % units because webpack was complaining

### Known Bugs
* on compilation time we get the below warning. i think this is caused by react-anime just not being updated in 3 years, so we might want to migrate to a different package that is currently being maintained. https://github.com/plus1tv/react-anime
```
WARNING in ./node_modules/react-anime/dist/react-anime.modern.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/ajtadeo/cookie-jar/node_modules/react-anime/dist/react-anime.modern.js.map' file: Error: ENOENT: no such file or directory, open '/Users/ajtadeo/cookie-jar/node_modules/react-anime/dist/react-anime.modern.js.map'
```

* Poppins font not applying to the website, instead the default Inter font is used + some other css differences. this might be caused by the weird change i mentioned earlier which made me remove all the semicolons in `App.scss`. for some reason the bulma styling is not being applied correctly.

### CODE REVIEW

**Comment by ajtadeo** [comment]
known warning on compilation:

```
WARNING in ./node_modules/react-anime/dist/react-anime.modern.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/ajtadeo/cookie-jar/node_modules/react-anime/dist/react-anime.modern.js.map' file: Error: ENOENT: no such file or directory, open '/Users/ajtadeo/cookie-jar/node_modules/react-anime/dist/react-anime.modern.js.map'
```

**Review by natalielordd — APPROVED** [review]
